src/cocaine/burlak/config.py

```diff
@@ -157,14 +157,16 @@ class Config(object):
                     'type': 'string',
                     'allowed': [
                         'tvm',
+                        'tvm2',
```

Comment:
if uppercase variants are allowed here, they should be allowed either by framework or converted to lowercase here or in framework.

Reply:
Framework understands both cases.